js/ui/pivot_grid/ui.pivot_grid.js

```diff
@@ -1656,7 +1657,9 @@ var PivotGrid = Widget.inherit({
         if(that._needDelayResizing(dataArea.getData()) && isFirstDrawing) {
```

Comment:
Why do you not check hasWindow in this case?

Reply:
You're right! I will fix it soon.